docker-compose.yml

```diff
@@ -18,3 +18,15 @@ services:
       - .:/editor:delegated
     ports:
       - "6006:6006"
+    environment:
+      - STORYBOOK_SIGNAL_URL=ws://signal:6000
```

Comment:
We need to pass storybook a link to our signaling service, it should look like this

Reply:
This looks like the link is an environment variable for the 'web' service - should it be an environment variable under  'signal'?

Reply:
No. This env variable is only relevant to the storybook build the web image is using.